src/schema/basic_classes.yaml

```diff
@@ -305,7 +305,11 @@ classes:
         structured_pattern:
           syntax: "^{id_nmdc_prefix}:(wfmag|wfmb|wfmgan|wfmgas|wfmsa|wfmp|wfmt|wfmtan|wfmtas|wfnom|wfrbt|wfrqc)-{id_shoulder}-{id_blade}{id_version}$|^{id_nmdc_prefix}:(omprc|dgms|dgns)-{id_shoulder}-{id_blade}$"
           interpolated: true
-
+      in_manifest:
```

Comment:
Can we add the `structured_pattern` to the global definition instead?  That way the `range` and the `structured_pattern` are stated in the same spot.